mpv.spec

```diff
@@ -95,15 +95,16 @@ waf configure \
     --disable-build-date \
     --enable-libmpv-shared \
     --enable-gpl3 \
-    --enable-sdl2
+    --enable-sdl2 \
+    --enable-encoding
```

Comment:
on my pc work

2016-04-13 0:00 GMT+06:00 Miro Hrončok <notifications@github.com>:

> In mpv.spec
> <https://github.com/rpmfusion/mpv/pull/12#discussion_r59423120>:
>
> > @@ -95,15 +95,16 @@ waf configure \
> >      --disable-build-date \
> >      --enable-libmpv-shared \
> >      --enable-gpl3 \
> > -    --enable-sdl2
> > +    --enable-sdl2 \
> > +    --enable-encoding
>
> Does it work without the patch?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/rpmfusion/mpv/pull/12/files/84afeb9ab76bc552bc6f30488e64e7e166953ada#r59423120>
>